src/DataViews/PresetsDataView.cpp

```diff
@@ -33,17 +34,25 @@ constexpr const char* kLoadableColumnName = "Loadable";
 constexpr const char* kPresetColumnName = "Preset";
 constexpr const char* kModulesColumnName = "Modules";
 constexpr const char* kHookedFunctionsColumnName = "Hooked Functions";
+constexpr const char* kDateModifiedColumnName = "Date Modified";
 
 constexpr const float kLoadableColumnWidth = 0.14f;
 constexpr const float kPresetColumnWidth = 0.34f;
-constexpr const float kModulesColumnWidth = 0.34f;
+constexpr const float kModulesColumnWidth = 0.20f;
 constexpr const float kHookedFunctionsColumnWidth = 0.16f;
+constexpr const float kDateModifiedColumnWidth = 0.16f;
 
 namespace {
 std::string GetLoadStateString(orbit_data_views::AppInterface* app, const PresetFile& preset) {
   orbit_data_views::PresetLoadState load_state = app->GetPresetLoadState(preset);
   return load_state.GetName();
 }
+
+std::string GetDateModifiedString(const PresetFile& preset) {
+  auto datetime_or_error = orbit_base::GetFormatedFileDateModified(preset.file_path());
+  if (datetime_or_error.has_error()) return "";
```

Comment:
also we lose the error message here, maybe add a ERROR() here?

Reply:
Added accordingly. Thanks!